Modules/TPC/src/ClusterVisualizer.cxx

```diff
@@ -194,6 +209,14 @@ void ClusterVisualizer::update(Trigger t, framework::ServiceRegistryRef)
   vecPtr = toVector(mCalDetCanvasVec.at(calDetIter));
   o2::tpc::painter::makeSummaryCanvases(calDet, int(mRanges[calDet.getName()].at(0)), mRanges[calDet.getName()].at(1), mRanges[calDet.getName()].at(2), false, &vecPtr);
   calDetIter++;
+
+  calDet = clusters.getOccupancy();
+  vecPtr = toVector(mCalDetCanvasVec.at(calDetIter));
+  o2::tpc::painter::makeSummaryCanvases(calDet, int(mRanges[calDet.getName()].at(0)), mRanges[calDet.getName()].at(1), mRanges[calDet.getName()].at(2), false, &vecPtr);
+  calDetIter++;
+  vecPtr = toVector(mCalDetCanvasVec.at(calDetIter));
+  makeRadialProfile(calDet, vecPtr.at(0), int(mRanges[calDet.getName()].at(0)), mRanges[calDet.getName()].at(1), mRanges[calDet.getName()].at(2));
+  calDetIter++;
```

Comment:
Here is a problem in the logic, since `calDet` is a __reference__ to the `NClusters` cal det. So all subsequent assignments will overwrite the `NClusters` calDet. Also, there is a lot of unnecessary repetitive code here. I propose to add a simple lambda and replace everything below
```c++
auto& clusters = clusterData->getClusters();
```
with
```c++
auto fillCanvases = [&calDetIter, this](const CalDet& calDet) {
  auto vecPtr = toVector(mCalDetCanvasVec.at(calDetIter++));
  const auto& ranges = mRanges[calDet.getName()];
  o2::tpc::painter::makeSummaryCanvases(calDet, int(ranges.at(0)), ranges.at(1), ranges.at(2), false, &vecPtr);
};

  fillCanvases(clusters.getNClusters());
  fillCanvases(clusters.getQMax());

  if (mIsClusters) {
    fillCanvases(clusters.getQTot());
    fillCanvases(clusters.getSigmaPad());
    fillCanvases(clusters.getSigmaTime());
  }

  fillCanvases(clusters.getTimeBin());
  fillCanvases(clusters.getOccupancy());

  makeRadialProfile(calDet, mCalDetCanvasVec.at(calDetIter++).at(0).get());
```

This should be much more compact and easier to read.